Rock.Messaging.RabbitMQ/IRabbitSessionConfiguration.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace Rock.Messaging.RabbitMQ
+{
+    public interface IRabbitSessionConfiguration
```

Comment:
Setters aren't necessary in a configuration interface.

Also, be sure to document all the properties and keep the line spacing consistent. I would also consider renaming vHost - what does RabbitMQ call it? Is it vHost or VirtualHost? I would name it consistent with the RabbitMQ docs.

re: validation - probably not a bad idea, but we're not sure where the best place for it to live is. You can probably table it for now (but document what an invalid config would look like).